en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md

```diff
@@ -1,361 +1,104 @@
+Before deploying WSO2 API Manager (WSO2 API-M), let's understand how the WSO2 API-M distributed deployment works.
+According to the recommended [deployment patterns]({{base_path}}/install-and-setup/setup/deployment-overview/#simple-scalable-deployment), a distributed deployment includes API-M server nodes that separately run the [API-M profiles]({{base_path}}/install-and-setup/setup/distributed-deployment/product-profiles). An API-M profile is a runtime instance that only runs specific components of the runtime.
 
-## API-M Components
-
-WSO2 API Manager uses the following main components:
+## API-M Profiles
 
-<a href="{{base_path}}/assets/img/setup-and-install/api-m-components.png"><img src="{{base_path}}/assets/img/setup-and-install/api-m-components.png" alt="api-m components"></a>
+Listed below are the different profiles available in WSO2 API Manager. 
 
 <table>
     <tr>
         <th>
-            Component
-        </th>
+            Profile
+        </td>
         <th>
             Description
-        </th>
-    </tr>
-    <tr>
-        <td>
-            Gateway
-        </td>
-        <td>
-            Responsible for securing, protecting, managing, and scaling API calls.
         </td>
     </tr>
     <tr>
         <td>
-            Key Manager
+            Gateway Profile
         </td>
         <td>
-            Responsible for all security and key-related operations.
+            <p>Only starts the components related to the API Gateway.</p>
+<p>Use this when the API Gateway acts as a worker node in a cluster. This profile starts the backend features for data processing and communicates with the management node.</p>
         </td>
     </tr>
     <tr>
         <td>
-            Traffic Manager
+            Control Plane Profile
         </td>
         <td>
-            Used to make a decision on throttling. It also works as an event hub for broadcasting controller events such as throttling events, block conditions, revoke token retrieval events, API events, API policy events, application events, application policy events, application keys events, subscription events, and subscription policy events.
+            Starts all the API-M components (Traffic Manager, Key Manager, Publisher, Developer Portal) excluding the Gateway.
         </td>
     </tr>
     <tr>
         <td>
-            Publisher
+            Traffic Manager Profile
         </td>
         <td>
-            Enables API providers to easily publish their APIs, share documentation, provision API keys, and gather feedback on API features, quality, and usage.
-        </td>
-    </tr>
-    <tr>
-        <td>
-            Developer Portal
-        </td>
-        <td>
-            Enables consumers to self-register, discover API functionality, subscribe to APIs, evaluate them, and interact with API publishers.
+            Only starts the Traffic Manager component.
         </td>
     </tr>
 </table>
 
-## Databases for API-M components
```

Comment:
Sectionn on "Databases required for each profiles" should not be dropped.

Reply:
Ack. 
I moved it to the 'Databases' section. Will keep it under 'Distributed Deployment' and link back and forth between the database doc and distributed deployment doc.

Reply:
Noticed that is moved to "en/docs/install-and-setup/setup/setting-up-databases/overview.md".  Is there a specific reason to move? Knowledge on dbs that each node needs should better to be at here AFAIU.

Reply:
> Ack.
> I moved it to the 'Databases' section. Will keep it under 'Distributed Deployment' and link back and forth between the database doc and distributed deployment doc.

Ack

Reply:
I thought.. we can link back from the 'Distributed Setup Overview" to the Databases section. However, I missed adding that link.. 
I think it is okay to do what you suggested and have links going both ways..